### ISSUE TITLE
Ablity to alias dip to point to a docker container?

### ISSUE THREAD

**Issue by butsjoh** [bibendi/dip#121]
Hi,

Thnx for releasing the latest (7.0) version today. I see that the compiled binary is not longer there and i guess that also means we cannot longer install dip with brew?

We can get along with installing the gem but the problematic part in that is that we will need to have ruby installed first on the dev machines and not all our people are ruby minded (frontend devs). Do you think it would be possible to run dip inside a container and would behave like it was installed on the host machine? From my understanding that will be hard cause did will execute docker stuff in the context of the container then and not the host.

Just want to explore the possible alternatives for the decision to not build a compiled binary.

**Comment by bibendi** [comment]
Hi, When I used Mac two years ago there was Ruby installed by default. Is it not true for today?
Anyway, installing Ruby is not a big deal, just type `brew install ruby`.
I think using Dip in a container will come many problems.

The reason was in the release notes https://github.com/bibendi/dip/releases/tag/v7.0.0

- I used Travis to build Mac binary, but Travis is dead.
- I want to use at least Ruby 2.5, but RubyPacker using 2.4 with a very old bundler
